packages/core/src/types.ts

```diff
@@ -641,8 +641,8 @@ export interface StateMachine<
   TContext,
   TStateSchema extends StateSchema,
   TEvent extends EventObject,
-  TState extends Typestate<TContext> = Typestate<TContext>
-> extends StateNode<TContext, TStateSchema, TEvent, TState> {
+  TTypestate extends Typestate<TContext> = any
```

Comment:
what kind of an effect this change might have on consuming applications? default has changed after all

Reply:
None